src/main/java/VirtualTrattoria.java

```diff
@@ -1,28 +1,44 @@
-import java.io.FileNotFoundException;
-import java.util.ArrayList;
-import java.util.Scanner;
+import Controller.*;
+import Service.ClientService;
+import Service.ClientServiceImpl;
+
+
 public class VirtualTrattoria {
 
     public static void main(String[] args) {
 
-        Scanner scanner = new Scanner(System.in);
+        // couldn't figure out how can I fix list reset
+        ClientService clientService = new ClientServiceImpl();
+
+        ClientController clientController = new ClientController();
+        clientController.setClientService(clientService); // couldn't figure out how can I fix list reset
```

Comment:
I would add clientService to ClientControler constructor instead of using a setter as controller cannot work correctly without a service in that case.